server-side has() can only check for custom perms (#1836)
clerk/clerk-docs: docs/_partials/has-warning.mdx

> [!WARNING]

> > When using `has()` **on the server-side** to check permissions, it only works with **custom permissions**, as [system permissions](/docs/organizations/roles-permissions#system-permissions) are not included in `auth.sessionClaims.orgs_permissions`. To check system permissions, verify the user's role instead.

Reviewer: ```suggestion
> Using `has()` **on the server-side** to check permissions works only with **custom permissions**, as [system permissions](/docs/organizations/roles-permissions#system-permissions) aren;t included in `auth.sessionClaims.orgs_permissions`. To check system permissions, you must verify the user's role instead.
```
